packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -84,19 +83,23 @@ class Checkbox extends SlotLabelMixin(
         }
 
         /* visually hidden */
-        ::slotted(input) {
+        [part='checkbox-wrapper'] ::slotted(input) {
           position: absolute;
           top: 0;
           left: 0;
+          right: 0;
+          width: 100%;
+          height: 100%;
           opacity: 0;
           cursor: inherit;
           margin: 0;
         }
       </style>
       <div part="container">
-        <div part="checkbox"></div>
-
-        <slot name="input"></slot>
+        <div part="checkbox-wrapper">
```

Comment:
We probably should remove `part` from here and then use `position: relative` to `src` styles.

Reply:
There is a new version that doesn't use `part`, but `id` instead. Does it still apply to the new version?